pkg/services/featuremgmt/registry.go

```diff
@@ -65,6 +65,11 @@ var (
 			State:       FeatureStateBeta,
 			Owner:       grafanaAppPlatformSquad,
 		},
+		{
+			Name:        "azLegacyTemplateVariables",
+			Description: "Azure Legacy Template Variables",
+			State:       FeatureStateBeta,
```

Comment:
```suggestion
			State:       FeatureStateBeta,
			Owner:       grafanaPartnerPluginsSquad,
```
and 

```go
	grafanaPartnerPluginsSquad                  codeowner = "@grafana/partner-plugins"
```

this one was added in https://github.com/grafana/grafana/pull/52584 - it looks like it belongs to partner plugins squad as its for the Azure Monitor datasource 

cc @aangelisc @vickyyyyyyy

Reply:
This flag can be removed now @ArturWierzbicki 😊